dbms/src/Flash/Mpp/ExchangeReceiver.h

```diff
@@ -36,8 +34,8 @@ struct ExchangeReceiverResult
     size_t call_index;
     String req_info;
     bool meet_error;
-    String error_msg;
     bool eof;
+    String error_msg;
```

Comment:
Define `error_msg` after `eof` so that `meet_error` and `eof` can share the same padding and minimal the struct size of `ExchangeReceiverResult`